action/steps/activate.sh

```diff
@@ -15,8 +15,13 @@ if [[ -n "$UNITY_LICENSE" ]]; then
   # Set the license file path
   FILE_PATH=UnityLicenseFile.ulf
 
-  # Copy license file from Github variables
-  echo "$UNITY_LICENSE" | tr -d '\r' > $FILE_PATH
+  if [[ -n "$UNITY_LICENSE" ]]; then
+    # Copy license file from Github variables
+    echo "$UNITY_LICENSE" | tr -d '\r' > $FILE_PATH
+  else
+    # Copy license file from file system
+    cat "$UNITY_LICENSE_PATH" | tr -d '\r' > $FILE_PATH
```

Comment:
This else implies that any other case is using a file path, which is not the case.

The variable should also correctly be called UNITY_LICENSE_FILE, as it is a file, not a path.